docs/blog/arcade/tilecode.md

```diff
@@ -0,0 +1,59 @@
+![tilecode banner](/static/blog/arcade/tilecode/banner.JPG)
+
+## TileCode: Design, Code, and Play Games on MakeCode Arcade Devices
+
+**Posted on August ???, 2020 by Thomas Ball and Stefania Druga**
+
+[TileCode](https://microsoft.github.io/tilecode) is a game creation app that allows you to design, code, and play video games directly on low-cost [Microsoft MakeCode Arcade gaming handhelds](https://arcade.makecode.com/hardware), as well as in the web browser.
+
+### From Board Games to Retro Video Games
+
+We draw inspiration from board games with pieces that can move from one square of the board to a nearby square. Board games like checkers (also known as draughts) have been played for thousands of years of human history.  These games are affordable, and their rules are generally simple to understand, but the game play can be very involving and rewarding. Over time, new games have been created by modifying the game rules of existing board games.
```

Comment:
Unless early primates or stegosaurus played checkers, "of human history" is implied.

Reply:
```suggestion
We draw inspiration from board games with pieces that can move from one square of the board to a nearby square. Board games like checkers (also known as draughts) have been played for thousands of years.  These games are affordable, and their rules are generally simple to understand, but the game play can be very engaging and rewarding. Over time, new games have been created by modifying the rules of existing board games.
```

Reply:
thanks, @ganicke